code/__DEFINES/traits.dm

```diff
@@ -930,3 +930,6 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 
 /// Makes a species be better/worse at defending against tackling depending on their tail's status
 #define TRAIT_TACKLING_TAILED_DEFENDER "tackling_tailed_defender"
+
+/// This target has recently been shot by a marksman coin and is very briefly immune to being hit by one again to prevent recusion
```

Comment:
```suggestion
/// This target has recently been shot by a marksman coin and is very briefly immune to being hit by one again to prevent recursion
```